src/odb/src/lefin/lefin.cpp

```diff
@@ -804,6 +804,24 @@ void lefin::layer(lefiLayer* layer)
     }
   }
 
+  if (layer->hasArraySpacing()) {
+    const bool is_long = layer->hasLongArray();
```

Comment:
Looks to be unused.

Reply:
that would be an oversight, I'll set that parameter